### DISCUSSION TITLE
Config form for data repository

### DISCUSSION THREAD

**Discussion by brockfanning** [open-sdg/open-sdg#1426] [Ideas]
**Is your feature request related to a problem? Please describe.**

There are configuration forms for both the site settings and the indicator settings, but not for the data settings. The data YAML config file still needs to be edit manually.

**Describe the solution that you need**

A configuration form for the data repository would help.

**Why do you need this feature?**

These config forms are self-documenting and make it easier to update the platform.

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin @AnnCorp @jwestw 
I've re-thought this, given the complexity of the data config forms. In particular, because parts of it consist of parameters for Python classes, and each Python class has a large number of possible parameters. As is, it would be very difficult and the form would be pretty confusing.

A middle-ground might be a form which has only the simple form settings, and simple choices between various "presets". For example, there could be a drop-down choice of presets for "Data input", and the choices could be "CSV files", "SDMX files", "SDMX API", etc. Similarly, there could be a drop-down choice of presets for "Metadata input", and the choices could be "YAML files", "Word files", "SDMX API", etc.

That would be a lot more user-friendly. Since the inputs are undergoing a lot of change right now, I'm wondering if it might be a good idea to postpone this until version 1.5.0, when we'll have a better idea of what is working best and what countries are using most.

**Comment by AnnCorp** [comment]
@brockfanning happy to push back to 1.5.0
